samples/suit/smp_transfer/README.rst

```diff
@@ -0,0 +1,393 @@
+.. _nrf54h_suit_sample:
+
+SUIT DFU: Firmware update on the nRF54H20 SoC
+#############################################
+
+.. contents::
+   :local:
+   :depth: 2
+
+The sample demonstrates how to update and boot the nRF54H20 System-on-Chip (SoC) using the Software Update for Internet of Things (SUIT) procedure on the Application and Radio Cores of the SoC.
+The update on the nRF54H20 SoC can be done over Bluetooth® Low Energy or UART.
+
+Requirements
+************
+
+The sample supports the following development kit:
+
+.. table-from-rows:: /includes/sample_board_rows.txt
+   :header: heading
+   :rows: nrf54h20dk_nrf54h20_cpuapp
+
+You need the nRF Connect Device Manager app for SUIT update over Bluetooth Low Energy:
+
+* `nRF Device Manager mobile app for Android`_
+  (The minimum required version is v1.9.)
```

Comment:
Due to ZCBOR incompatibilities, we will need a new version of the app(s).

Reply:
Okay, do we know yet if it is the latest version of the apps/ a specific version no.?

Reply:
It is still in development. The next version will be 1.10.

Reply:
Do we know when it will be released? If not, perhaps we can merge this PR for now, and I can create a task to update this when the next version is out. Doc freeze is 6 June, so we have until then.